src/API/Init.php

```diff
@@ -74,7 +74,7 @@ public function rest_api_init() {
 			'Automattic\WooCommerce\Admin\API\Themes',
 		);
 
-		if ( Loader::is_onboarding_enabled() ) {
+		if ( Loader::is_onboarding_enabled() || Loader::is_feature_enabled( 'shipping-label-banner' ) ) {
```

Comment:
We will enable the onboarding REST end points for shipping-label-banner as well. By doing this, we probably no longer need to set onboarding settings in `component_settings` (`src/Features/ShippingLabelBanner.php`) anymore.